docker-compose.yml

```diff
@@ -33,9 +33,13 @@ services:
       WORKERS_COUNT: 2
   redis:
     image: redis:3.0-alpine
+    ports:
+      - 6379:6379
     restart: unless-stopped
   postgres:
     image: postgres:9.5.6-alpine
+    ports:
+      - 5432:5432
```

Comment:
While exposing the ports is useful, I don't think we should include this in the default configuration as it might conflict with other configurations the user might have.